### DISCUSSION TITLE
Voting-only node in 5-node cluster

### DISCUSSION THREAD

**Discussion by Ricaz** [rabbitmq/rabbitmq-server#10009] [Questions]
Hi,

I have an ELK-like cluster of 3 zones. Zone A and B do all the heavy lifting, and C is essentially just there to avoid a split-brain scenario if one zone goes down. I want to avoid doing any real work in zone C.

Currently, it has 2 RabbitMQ nodes each in zones A and B, but none in C. I want to add another RabbitMQ node in C, but I don't want it to do any real work. I just want to ensure consensus in case of failure in A or B.

All zones are geographically separated but on the same virtual network.

Is this possible, and if so, how?

**Comment by michaelklishin** [comment]
@Ricaz there are no "voting only" queue or stream replicas in RabbitMQ. There are no "special" nodes in RabbitMQ in general.

Three nodes is a [required minimum](https://rabbitmq.com/clustering.html#node-count) for quorum queues and streams, Khepri (the new schema data store) starting with 3.13, and everything starting with 4.0.

There are no plans to introduce witness replicas or "tie breaking voter" nodes. Every Raft-based feature technically has a concept of non-voting members but it is a temporary state, not a permanent role https://github.com/rabbitmq/ra/pull/375.
